src/store/currentTodoReducer.ts

```diff
@@ -0,0 +1,31 @@
+import { CurrentTodoReducerTypes } from '../types/CurrentTodoReducerTypes';
+
+export const actions = {
+  // eslint-disable-next-line max-len
+  setTodo: (todoId: number) => ({ type: CurrentTodoReducerTypes.SET_TODO, payload: todoId }),
+};
+
+type CurrentTodoAction = {
+  type: CurrentTodoReducerTypes.SET_TODO,
+  payload: number,
+};
+
+export const selectors = {
+  setTodo: (todoId: number) => todoId,
+};
+
+const initialState = 0;
+
+const loadingReducer = (
+  state = initialState,
+  action: CurrentTodoAction,
+): number => {
+  switch (action.type) {
+    case CurrentTodoReducerTypes.SET_TODO:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+export default loadingReducer;
```

Comment:
create one reducer for Todos you don't need separate reducers for current and all todos